package.json

```diff
@@ -46,6 +46,7 @@
     "test-watch": "karma start --auto-watch --no-single-run"
   },
   "dependencies": {
+    "connect-modrewrite": "^0.9.0",
```

Comment:
I'm not really sure what to do except make sure you've merged develop into your branch. It would give you a merge conflict if you did and you would accept the develop code. If you've already kept up-to-date with develop, then I have no idea what you did. The fix for this would not be a code change.